packages/atomic/src/utils/utils.ts

```diff
@@ -56,13 +56,28 @@ export function parseHTML(string: string) {
   return new window.DOMParser().parseFromString(string, 'text/html');
 }
 
+export function isElementNode(node: Node): node is Element {
```

Comment:
I think quite a bit of code in this PR could benefit from good ol' unit test. Since a lot is pure function !